src/app/shared/services/auth.service.ts

```diff
@@ -12,6 +12,9 @@ import { RestApiHelperService } from 'app/shared/utils/rest-api-helper.service';
 @Injectable()
 export class AuthService {
     private restangular: Restangular;
+    private email: string;
```

Comment:
Why are we keeping separate variables for these fields? Shouldn't we keep a JSON?

Reply:
done.